src/transformers/models/led/modeling_tf_led.py

```diff
@@ -1862,28 +1862,16 @@ def call(
         hidden_states = inputs["inputs_embeds"]
 
         # [bsz, seq_len] -> [bsz, 1, tgt_seq_len, src_seq_len]
-        combined_attention_mask = None
         if input_shape[-1] > 1:
             combined_attention_mask = _make_causal_mask(input_shape, past_key_values_length=past_key_values_length)
         else:
             combined_attention_mask = _expand_mask(
                 tf.ones((input_shape[0], input_shape[1] + past_key_values_length)), tgt_len=input_shape[-1]
             )
 
-        if inputs["attention_mask"] is None and inputs["input_ids"] is not None and input_shape[-1] > 1:
```

Comment:
I wrongly copied this from the old Bart templates. Those lines previously automatically create a correct attention mask in case there are pad_tokens. However, we do not support this behavior in the PyTorch version of LED and should not support it. In case some inputs should be padded the tokenizers should take care of masking the inputs.